index.html

```diff
@@ -0,0 +1,322 @@
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div id="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+      </section>
+      <section CLASS="sidebar2">
```

Comment:
Please use lower casing for your class name. "class" not "CLASS"

Reply:
- [ ] As requested above, it is advisable to give a reason behind a request. Because it will give clarity on what to improve 🌞 Please you can edit your inline comment to achieve that 🙇🏽